src/js/views/tweet.js

```diff
@@ -12,7 +13,8 @@ class Tweet extends React.Component {
       React.createElement('div', {className: 'content'},
         React.createElement('div', {className: 'date'}, formattedDate),
         React.createElement('a', {className: 'link', target: '_blank', href: `https://twitter.com/${user.twitterHandle}/status/${tweet.id}`},
-          React.createElement('div', {className: 'msg'}, `${tweet.message}`))
+          React.createElement('div', {className: 'msg'}, `${tweet.message}`)),
+        this.props.isHomeTimeline ? React.createElement('button', {className: 'replyButton', onClick: () => {ReplyListener.publish(tweet)}}, React.createElement('i', {className: 'fas fa-reply fa-sm'})) : null
```

Comment:
this component should not need to know about "isHometimeline". Use better variable names